cms_frontend/src/pages/Dashboard.tsx

```diff
@@ -19,6 +20,19 @@ const Directory = styled.h3`
 const Dashboard: React.FC = () => {
   const [dir, setDir] = useState("root" as FolderName);
 
+  // Modal state handler
+  const [open, setOpen] = React.useState(false);
+
+  // Modal opener
+  const handleOpen = () => {

```

Comment:
maybe name it more specificically
`handleModalOpen` might be good, since hanyuan's PR includes alot of functions in the dashboard

Reply:
Will update. Thanks